workflow-controller/controller-service/src/main/java/io/hyscale/controller/invoker/StatusComponentInvoker.java

```diff
@@ -58,6 +61,14 @@
     @Autowired
     private TroubleshootService troubleshootService;
     
+    @Autowired
+    private K8SClusterValidatorHook k8SClusterValidatorHook;
+
+    @PostConstruct
+    public void init() {
+        super.addHook(k8SClusterValidatorHook);
+    }
```

Comment:
Should be for logs command also

Reply:
In case of Logs command we are directly using it in LoggerUtility and not invoker, the utility is itself validating the cluster hence didn't feel the need to add validation hook.
Should I add a manual call to k8SClusterValidatorHook.preHook in loggerUtility?

Reply:
Good, then in such case it is not required .